docs/running-headscale-container.md

```diff
@@ -22,13 +22,9 @@ mkdir -p ./headscale/config
 cd ./headscale
 ```
 
-2. Create an empty SQlite datebase in the headscale directory:
+2. **(Strongly Recommended)** Download a copy of the [example configuration](https://github.com/juanfont/headscale/blob/main/config-example.yaml) from the headscale repository.
 
-```shell
-touch ./config/db.sqlite
-```
-
-3. **(Strongly Recommended)** Download a copy of the [example configuration](https://github.com/juanfont/headscale/blob/main/config-example.yaml) from the headscale repository.
+The following instructions are for running against `headscale/headscale:latest`, if you are using an older version, you will want to make sure you're getting the correct version of a default configuration file from the [tags](https://github.com/juanfont/headscale/tags) matching your version.
```

Comment:
There are [admonition extension](https://squidfunk.github.io/mkdocs-material/reference/admonitions/#supported-types) for such cases.

```
!!! warning "Strongly Recommended"
    
    Download a copy of the [example configuration](https://github.com/juanfont/headscale/blob/main/config-example.yaml) from the headscale repository.
```

Reply:
While I make changes like these by re-basing on `main`, and adding those changes, I cannot validate the changes since `make serve` is not offered in the `Makefile` after a rebase on `main` (which is [the validation step mentioned here](https://github.com/juanfont/headscale/pull/1575)). You can see those changes here in this branch if you'd like to take a look: https://github.com/canardleteer/headscale/blob/fix-docker-documentation-testing/docs/running-headscale-container.md

It does appear, than since my contribution, at least some of the changes proposed were added to the repository.

If those changes are fine as is, I can add them to this MR.